build_tools/cmake/iree_copts.cmake

```
@@ -306,6 +306,14 @@ else()
   # Android provides its own pthreads support with no linking required.
 endif()
 
+if(ANDROID)
+  # logging.h on Android needs llog to link in Android logging. 
+  iree_select_compiler_opts(_IREE_LOGGING_LINKOPTS
+    CLANG_OR_GCC
+    "-llog"
```

style nit: match indentation from the previous code block

```suggestion
    CLANG_OR_GCC
      "-llog"
```